rdt-open-reader/src/main/java/io/ona/rdt/activity/PatientRegisterActivity.java

```diff
@@ -209,6 +212,14 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                 return true;
             }
         });
+
+        long lastSyncTimestamp = RDTApplication.getInstance().getContext().allSharedPreferences().getPreferences().getLong(CovidConstants.SyncPreference.LAST_SYNC_DATE, 0);
+        if (lastSyncTimestamp > 0) {
+            Date lastSyncDate = new Date(lastSyncTimestamp);
+            String lblSync = getString(R.string.drawer_menu_item_sync) + ": %1$s";
```

Comment:
Define this place holder in the strings.xml

Reply:
done